pkg/controller/certificaterequests/sync.go

```diff
@@ -76,13 +73,19 @@ func (c *Controller) Sync(ctx context.Context, cr *v1alpha1.CertificateRequest)
 
 	issuerObj, err := c.helper.GetGenericIssuer(crCopy.Spec.IssuerRef, crCopy.Namespace)
 	if k8sErrors.IsNotFound(err) {
-		c.recorder.Eventf(crCopy, corev1.EventTypeWarning, errorIssuerNotFound, err.Error())
+		apiutil.SetCertificateRequestCondition(crCopy, v1alpha1.CertificateRequestConditionReady,
+			v1alpha1.ConditionFalse, errorPending, "Referenced Issuer not found")
```

Comment:
Can we replace `Issuer` here with `issuerRef(crCopy.Spec.IssuerRef.Kind)` (i.e. the `issuerRef` function can be found in the `pkg/controller/certificates` package, and can just be copied across). Should make this clearer to users 😄